src/types/TodoCondition.ts

```diff
@@ -0,0 +1,6 @@
+export enum TodoCondition {
+  neutral,
+  saving,
+  editing,
+  deleting,
```

Comment:
Enum values should be in PascalCase. Consider renaming 'deleting' to 'Deleting'.